config/v1/types_securityprofile.go

```diff
@@ -0,0 +1,406 @@
+package v1
+
+// TLSSecurityProfile defines the schema for a TLS security profile. This object
+// is used by operators to apply TLS security settings to operands.
+// +union
+type TLSSecurityProfile struct {
+	// type is one of Old, Intermediate, Modern or Custom. Custom provides
+	// the ability to specify individual TLS security profile parameters.
+	// Old, Intermediate and Modern are TLS security profiles based on:
+	//
+	// https://wiki.mozilla.org/Security/Server_Side_TLS#Recommended_configurations
+	//
+	// The profiles are intent based, so they may change over time as new ciphers are developed and existing ciphers
+	// are found to be insecure.  Depending on precisely which ciphers are available to a process, the list may be
+	// reduced.
+	//
+	// +unionDiscriminator
+	// +optional
+	Type TLSProfileType `json:"type"`
+	// old is a TLS security profile based on:
+	//
+	// https://wiki.mozilla.org/Security/Server_Side_TLS#Old_backward_compatibility
+	//
+	// and looks like this (yaml):
+	//
+	//   ciphers:
```

Comment:
@deads2k the cipher suites have been updated in the Mozilla reference. I suggest recreating the example ciphers and profile ciphers based on the new list. Additionally, the ciphers are now grouped based on TLS version. How do you suggest we handle the cipher<> TLS version association?